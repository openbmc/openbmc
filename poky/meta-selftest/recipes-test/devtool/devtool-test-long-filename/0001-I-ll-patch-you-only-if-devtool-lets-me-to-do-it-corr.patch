diff --git a/patch-me b/patch-me
index a20b29a..5e35d1b 100644
--- a/patch-me
+++ b/patch-me
@@ -1 +1 @@
-please
+NO
