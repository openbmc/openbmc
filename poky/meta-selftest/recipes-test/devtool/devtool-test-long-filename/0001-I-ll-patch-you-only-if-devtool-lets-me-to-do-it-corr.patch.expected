From 45ba3d107ea60777a6b6e134fd00fe5009749177 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Thu, 28 May 2020 02:03:39 +0200
Subject: [PATCH] meta-selftest: add test for .patch file with long filename

---
 patch-me | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/patch-me b/patch-me
index a20b29a..5e35d1b 100644
--- a/patch-me
+++ b/patch-me
@@ -1 +1 @@
-please
+NO
